sql-bricks.js

```diff
@@ -29,7 +29,7 @@
       if (!opts.values.length)
         return match;
 
-      var ix = capture ? parseInt(capture, 10) : opts.value_ix++;
+      var ix = arguments.length > 3 ? parseInt(capture, 10) : opts.value_ix++;

```

Comment:
@vitalif: Nice catch! To make this a little clearer, I think the function should be defined as:

```javascript
// replacer(match, [capture1, capture2, ...,] offset, string)
function replacer() {
```

And this line would then be:

```javascript
var ix = arguments.length > 3 ? parseInt(arguments[1], 10) : opts.value_ix++;
```